app/src/main/java/com/example/hsh/homesweethome/furniture/view/FurnitureViewHolder.java

```diff
@@ -17,7 +17,7 @@
 
 public class FurnitureViewHolder extends RecyclerView.ViewHolder implements RecyclerViewHolderFurniture {
 
-    TextView furniture_title, furniture_brand, furniture_price;
+    TextView furniture_title, furniture_brand, furniture_price, furniture_in_store;
```

Comment:
are there some bootstrap badge that we can use instead of a text field? The logic is there but perhaps can spice up the UI a little

Reply:
Perhaps in the future Noted.